manifests/init.pp

```diff
@@ -41,7 +41,7 @@
 
   # this warning is never reached. If Puppet < 4 is used with tis module,
   # Puppet fail with error about data type like "Syntax error at 'String'; expected ')'"
-  if versioncmp($::puppetversion, '4.0.0') == -1 {
+  if versioncmp($facts['puppetversion'], '4.0.0') == -1 {
```

Comment:
Time to remove this altogether? Version 2.0 dropping puppet 3 was out a long time ago now.

Reply:
Yes i planed to do this in an other PR. But if you agree we can do it now.

Reply:
I'd be fine with that :+1: